pkg/policies/outside/outside.go

```diff
@@ -293,12 +321,28 @@ func getUsers(ctx context.Context, r repositories, owner, repo, perm,
 	var rv []string
 	for _, u := range users {
 		if u.GetPermissions()[perm] {
-			rv = append(rv, u.GetLogin())
+			if !exemptions.isExempt(repo, u.GetLogin(), perm) {
+				rv = append(rv, u.GetLogin())
+			}
 		}
 	}
 	return rv, nil
 }
 
+func (ee OutsideExemptions) isExempt(repo, user, access string) bool {
```

Comment:
In general allstar is coded as procedural and not object oriented. In this case there is no benefit to this being a function on the type, it could just be a parameter.